pyswarms/single/global_best.py

```diff
@@ -153,10 +157,14 @@ def optimize(self, objective_func, iters, print_step=1, verbose=1):
         tuple
             the global best cost and the global best position.
         """
+
+        cli_print("Arguments Passed to Objective Function: \nargs: %s \nkwargs: %s\n" % (args, kwargs),
+                  verbose, 2, logger=self.logger)
+
```

Comment:
This is good, keep this. I'd suggest using:

```
cli_print("Arguments passed to objective function: \nargs: %s \nkwargs: %s\n" % (*args, **kwargs),
                  verbose, 2, logger=self.logger)
```

Reply:
It appears that using `**` outside of a function call throws an `IndexError`.  The Python documentation around this uses them without the decorators in the outside of the function call.
https://docs.python.org/dev/tutorial/controlflow.html#more-on-defining-functions
Thoughts?

Reply:
Whoops. Yes, you are correct, just remove `**` and it should print. One last Python 3.X thing,  perhaps a `.format()` is better?

```python
name = 'ljvmiranda921'
print("Hi my name is: {}".format(name))
```